Microsoft.Toolkit.HighPerformance/Enumerables/ReadOnlySpanEnumerable{T}.cs

```diff
@@ -42,8 +43,9 @@ public ReadOnlySpanEnumerable(ReadOnlySpan<T> span)
         /// Implements the duck-typed <see cref="IEnumerable{T}.GetEnumerator"/> method.
         /// </summary>
         /// <returns>An <see cref="ReadOnlySpanEnumerable{T}"/> instance targeting the current <see cref="ReadOnlySpan{T}"/> value.</returns>
+        [Pure]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public ReadOnlySpanEnumerable<T> GetEnumerator() => this;
+        public readonly ReadOnlySpanEnumerable<T> GetEnumerator() => this;
```

Comment:
Are these unrelated? Should they be a separate PR? ;)

Also, are these breaking changes or not really as it was a readonly type already? I'm guessing this is more of a compiler hint that you'd be doing something wrong?

Reply:
Yup, sorry got carried away 😅
Reverted them in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3346/commits/cd70d0dbfa80ea987a3bea4b0a0e676390bd6ed6, will move those to https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3351.